### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by Saranshmalik**
New changes : 
- Added support for push-notifications #2 
You can now register your notification token with zendesk to enable notifications. Handling and displaying the notifications is to be done at app level, the package won't support that

- Added typescript support to the module

### CODE REVIEW

**Review by stewartmcgown — APPROVED** [review]
Concise and helpful PR!

**Comment by Saranshmalik** [comment]
Merging

File: android/src/main/java/com/saranshmalik/rnzendeskchat/RNZendeskChat.java [hunk 1]
```diff
@@ -156,6 +157,7 @@ public void startChat(ReadableMap options) {
         Providers providers = Chat.INSTANCE.providers();
         setUserIdentity(options);
         setVisitorInfo(options);
+        setUserIdentity(options);
```
Comment (tarasywe) [inline android/src/main/java/com/saranshmalik/rnzendeskchat/RNZendeskChat.java:160]: what is the reason of call the same method twice?

Comment (saransh-malik) [reply android/src/main/java/com/saranshmalik/rnzendeskchat/RNZendeskChat.java:160]: Probably a bad merge, thanks for the point out. Will remove this.

**Pull request merged**